ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -654,20 +655,37 @@ def verify_storage_clients_page(self):
         """
         Verify storage clients page in UI
 
-        Returns:
-        StorageClients: Storage Clients page object
-
         """
         self.refresh_web_console()
-        storage_client_obj = self.nav_to_storageclients_page()
+        self.nav_to_storageclients_page()
         strings_storage_clients_tab = ["Storage clients", "Name"]
         self.verify_page_contain_strings(
             strings_on_page=strings_storage_clients_tab, page_name="storage clients"
         )
         self.do_click(
-            self.validation_loc["generate_client_onboarding_token_button"],
+            self.storage_clients_loc["generate_client_onboarding_ticket"],
             enable_screenshot=True,
         )
+        # Starting from ODF 4.17 custom storage amount select option is available
+        if self.ocs_version_semantic >= version.VERSION_4_17:
+            select_storage_quota_window = [
+                "Add storage capacity for the client cluster to consume from the provider cluster."
+                "Storage quota: Unlimited",
+                "Unlimited",
+                "Custom",
+            ]
+            self.verify_page_contain_strings(
+                strings_on_page=select_storage_quota_window,
+                page_name="client_onboarding_token_page",
+            )
+            # Check default storage quota selected as 'Unlimited'
+            assert wait_for_element_to_be_visible(
+                self.validation_loc["storage_quota_unlimited"]
+            ).is_selected(), "Default value unlimited quota is not selected"
+
+            # Take screenshot
+            self.take_screenshot()
```

Comment:
pls use description param, this way we will identify screenshot by name in the UI logs dir 
something like 
self.take_screenshot("verify_storage_clients_page_done")